### PR TITLE
Simplify and improve formatting/metadata in events.js

### PR DESCRIPTION
**Pull request by stevenleggdfe**

### CODE REVIEW

File: includes/events.js [hunk 1]
```diff
@@ -1,63 +1,54 @@
 module.exports = (params) => {
-  var namespaceFilterSql = '';
```
Comment (stevenleggdfe) [inline includes/events.js:2]: I've moved this logic into inline JS in the main body of the SQL

File: includes/events.js [hunk 2]
```diff
@@ -1,63 +1,54 @@
...
-  if (params.bqEventsTableNameSpace) {
-    namespaceFilterSql = `AND namespace = '${params.bqEventsTableNameSpace}'`;
-  }
-  return publish("events_" + params.eventSourceName, {
```
Comment (stevenleggdfe) [inline includes/events.js:6]: The only change to this part is extra metadata on the DATA field (line 35/31) - the rest is just autoformatted indentation changes

File: includes/events.js [hunk 3]
```diff
@@ -1,63 +1,54 @@
...
+            device_category: "The category of device used to cause this event - desktop, mobile, bot or unknown.",
+            browser_name: "The name of the browser used to cause this event.",
+            browser_version: "The version of the browser used to cause this event.",
+            operating_system_name: "The name of the operating system used to cause this event.",
+            operating_system_vendor: "The vendor of the operating system used to cause this event.",
+            operating_system_version: "The version of the operating system used to cause this event."
+        }
+    }).query(ctx => `WITH
+  earliest_web_request_event_for_request AS (
   SELECT
```
Comment (stevenleggdfe) [inline includes/events.js:49]: Completely unnecessary CTE that I've replaced with an analytics ROW_NUMBER() function in line 70 in the new file

File: includes/events.js [hunk 4]
```diff
@@ -1,63 +1,54 @@
...
+            anonymised_user_agent_and_ip: "One way hash of a combination of the user's IP address and user agent. Can be used to identify the user anonymously, even when user_id is not set. Cannot be used to identify the user over a time period of longer than about a month, because of IP address changes and browser updates.",
+            device_category: "The category of device used to cause this event - desktop, mobile, bot or unknown.",
+            browser_name: "The name of the browser used to cause this event.",
+            browser_version: "The version of the browser used to cause this event.",
+            operating_system_name: "The name of the operating system used to cause this event.",
+            operating_system_vendor: "The vendor of the operating system used to cause this event.",
+            operating_system_version: "The version of the operating system used to cause this event."
+        }
+    }).query(ctx => `WITH
+  earliest_web_request_event_for_request AS (
```
Comment (stevenleggdfe) [inline includes/events.js:48]: Renamed CTE from earliest_event_for_web_request - not a change in logic but I think clearer

**Review by sophiereynolds — APPROVED** [review]

**Pull request merged**